packages/integration_test/lib/fix_data/README.md

```diff
@@ -0,0 +1,45 @@
+## Directory contents
```

Comment:
This readme (and the template.yaml) should be the same here, for the flutter framework, and for flutter_test, right? I wonder, if it would make sense to maintain it only once (maybe in the framework) and have the other packages just link to that in their readme? There's not much value in repeating all this information and the policies should be consistent between packages, anyways...